test/integration/moderation.js

```diff
@@ -389,6 +389,8 @@ describe('A user with a channel shadow ban', function () {
 
 describe('show ban status on member', function () {
 	const guyonID = `guyon-${uuidv4()}`;
+	const user1ID = `user1-${uuidv4()}`;
```

Comment:
what's the purpose of these additional userids here, they don't seem to be used anywhere

Reply:
This IDs are used in the tests for backwards compatibility [here](https://github.com/GetStream/stream-chat-js/pull/496/files#diff-d94fa608cd42452bb60618acc150397dd1e93ad02319fb4850c09cdcd32c2967R410)

Reply:
sorry, I missed that!